src/stories/Library/Icons/icon-favourite/icon-favourite.scss

```diff
@@ -2,6 +2,9 @@
   cursor: pointer;
   height: 24px;
   width: 24px;
+  color: $c-identity-primary;
+  stroke: $c-identity-primary;
+  fill: none;
```

Comment:
Please remove this change to the base favourite icon.

As discussed in Zulip the default design if the icon should not be changed.